### DISCUSSION TITLE
Would bulk inserts (CSV,JSON) via a file upload through the API be useful to you?

### DISCUSSION THREAD

**Discussion by thadguidry** [9tigerio/db2rest#153] [Polls]
Essentially this would act like a `COPY FROM`.
Ref: https://www.postgresql.org/docs/16/sql-copy.html
Here's a bit from the reference, but please consult it for available parameters.
> COPY FROM copies data from a file to a table (appending the data to whatever is in the table already).
> Each field in the file is inserted, in order, into the specified column. Table columns not specified in the COPY FROM column list will receive their default values.

> **column_name**
An optional list of columns to be copied. If no column list is specified, all columns of the table except generated columns will be copied.

>COPY FROM can be used with plain, foreign, or partitioned tables or with views that have INSTEAD OF INSERT triggers.

**Comment by kdhrubo** [comment]
Someone voted yes on this one 👍

**Comment by MichalisDBA** [comment]
Yep, +1 from me. I have use cases that i want to use curl to make a GET request to an API endpoint, get the json data and pipe it to db2rest API endpoint with a POST request to make bulk inserts to a db table.

**Reply by kdhrubo** [reply]
This will be available soon once the refactoring settles down. Tentatively release 0.3.5
